gpt4all-chat/chatlistmodel.h

```diff
@@ -192,13 +192,8 @@ class ChatListModel : public QAbstractListModel
 
     int count() const { return m_chats.size(); }
 
-    void clearChats() {
-        m_newChat = nullptr;
```

Comment:
Why are these being removed?

Reply:
clearChats, which I added in #1890, is being replaced by destroyChats, below. See the linked issue for why this is important.